docs/UserGuide.md

```diff
@@ -262,6 +262,22 @@ Redoes a command that the user has undone previously.
 
 Format: `redo`
 
+### Recall previous commands
+
+CCA Leaders can quickly recall previous commands and edit it slightly, allowing users to mark attendance for a group of members quickly.
```

Comment:
I think we should not limit the context of recalling commands just to "marking attendance for a group of members". Since recalling commands can be used for editing other types of commands, putting this in the command description may mislead users to thinking that it can only be used for one specific context.

Reply:
Ah yes that's true, what do you think about changing to the following

```
CCA Leaders can quickly recall previously entered commands for convenience. Example use cases include marking attendance for a group of members, editing slightly wrong details and more.
```

Reply:
Sounds good!